webflux5-sleuth/src/main/java/brave/example/Frontend.java

```diff
@@ -0,0 +1,34 @@
+package brave.example;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.reactive.function.client.WebClient;
+import reactor.core.publisher.Mono;
+
+@EnableAutoConfiguration
+@RestController
+@CrossOrigin // So that javascript can be hosted elsewhere
```

Comment:
Is there javascript?

Reply:
we used to integrate brave-webmvc-example with zipkin-js-example, but we can remove this here and in the other examples.. or maybe change the comment to say

"in case this is integrated with zipkin-js-example"

because if for example, you open index.html from filesystem, it would error.

Reply:
I think I'll remove it..